spec/support/user_helper.rb

```diff
@@ -28,5 +28,5 @@ def register_user
   fill_in :user_email, with: 'email@gmail.com'
   fill_in :user_password, with: 'password'
   fill_in :user_password_confirmation, with: 'password'
-  click_button 'Регистрация'
+  click_button 'Зарегистрироваться'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.